### PR TITLE
Remove slot size mapping from data attrs

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Remove size mapping data attributes from ad slots.

Some functionality requires a way to update size mappings that depended on the data attributes, that's now updated.

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (Done https://github.com/guardian/dotcom-rendering/pull/5543)

## What is the value of this and can you measure success?
They are redundant now, size mappings are now loaded from [commercial-core](https://github.com/guardian/commercial-core/blob/main/src/ad-sizes.ts#L143) as of this frontend PR https://github.com/guardian/frontend/pull/25124

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.ts [hunk 1]
```diff
@@ -48,7 +50,19 @@ const fillAdvertSlots = async (): Promise<void> => {
...
-		.map((adSlot) => new Advert(adSlot));
+		.map((adSlot) => {
+			let additionalSizes = {};
+
+			if (
+				window.guardian.config.page.contentType === 'Gallery' &&
+				adSlot.dataset.name?.includes('inline')
+			) {
+				additionalSizes = {
+					desktop: [adSizes.billboard, createAdSize(900, 250)],
```
Comment (Jakeii) [inline static/src/javascripts/projects/commercial/modules/dfp/fill-advert-slots.ts:61]: Removed from https://github.com/guardian/frontend/pull/25318/files#diff-bd054de0f93b3c5974cc80b7f32c6ef80f30ad70bc5648cce21983b7a29ba165L18, now needs to be added at advert initilization

File: static/src/javascripts/projects/commercial/modules/comment-adverts.ts [hunk 2]
```diff
@@ -1,6 +1,4 @@
-import type { SizeKeys } from '@guardian/commercial-core';
-import { adSizes, createAdSize, createAdSlot } from '@guardian/commercial-core';
-import { toGoogleTagSize } from 'common/modules/commercial/lib/googletag-ad-size';
+import { adSizes, createAdSlot } from '@guardian/commercial-core';
```
Comment (Jakeii) [inline static/src/javascripts/projects/commercial/modules/comment-adverts.ts:1]: Not sure if this module is hanging around (as it doesn't currently work), but have updated it to no longer use data-attributes to upgrade the slot and use a new method on the `Advert` class `updateSizeMapping`.

Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/comment-adverts.ts:1]: I think we kept this as we want to apply this logic to DCR but just haven't got to the ticket.

**Review by arelra — APPROVED** [review]
Great work 🎉 !

File: static/src/javascripts/projects/commercial/modules/comment-adverts.ts (hunk 2 above)
Comment (Jakeii) [reply static/src/javascripts/projects/commercial/modules/comment-adverts.ts:1]: sweet, yeah I had an inkling!

**Pull request merged**